docs/src/components/VersionsTable.tsx

```diff
@@ -0,0 +1,201 @@
+import {
+  ResponsiveTable,
+  ResponsiveTableCell,
+} from '@/components/ResponsiveTable';
+import {
+  Tabs,
+  TableCell,
+  TableBody,
+  TableHead,
+  TableRow,
+} from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+import {
+  Framework,
+  PREV_MAJOR_VERSIONS,
+  CURRENT_MAJOR_VERSIONS,
+} from '../data/frameworks';
+
+export const VersionsTable = ({ framework }) => {
+  const {
+    query: { platform = 'react' },
+  } = useRouter();
+
+  if (!framework) {
+    framework = platform as Framework;
+  }
+
+  const latestVersion = CURRENT_MAJOR_VERSIONS[framework].toString();
+  const prevVersion = PREV_MAJOR_VERSIONS[framework].toString();
+
+  return (
+    <Tabs.Container defaultValue={latestVersion}>
+      <Tabs.List>
+        <Tabs.Item value={latestVersion}>
+          ui-{framework} v{latestVersion} (latest)
+        </Tabs.Item>
+        <Tabs.Item value={prevVersion}>
+          ui-{framework} v{prevVersion}
+        </Tabs.Item>
+      </Tabs.List>
+      <Tabs.Panel value={latestVersion}>
+        <ResponsiveTable labelWidth="10rem">
+          <TableHead>
+            <TableRow>
+              <TableCell as="th">Function Call</TableCell>
+              <TableCell as="th">Override Name</TableCell>
+              <TableCell as="th">formData Properties</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <ResponsiveTableCell label="Function Call">
+                <code>signUp</code>
+              </ResponsiveTableCell>
+              <ResponsiveTableCell label="Override Name">
+                <code>handleSignUp</code>
+              </ResponsiveTableCell>
+              <ResponsiveTableCell label="formData Properties">
+                <code>{'{(username, password)}'}</code>
+              </ResponsiveTableCell>
+            </TableRow>
+            <TableRow>
+              <ResponsiveTableCell label="Function Call">
+                <code>signIn</code>
+              </ResponsiveTableCell>
+              <ResponsiveTableCell label="Override Name">
+                <code>handleSignIn</code>
+              </ResponsiveTableCell>
+              <ResponsiveTableCell label="formData Properties">
+                <code>{'{(username, password)}'}</code>
```

Comment:
Think the component this replaces used backticks instead of single quotes?

Reply:
I used `<code>` instead of backticks bc the previous way doesn't work in the .tsx file. The single quotes don't show up it's just to make it a string